site/cds_rdm/ldap/client.py

```diff
@@ -45,7 +45,7 @@ class LdapClient(object):
 
     def __init__(self, ldap_url=None):
         """Initialize ldap connection."""
-        ldap_url = ldap_url or current_app.config["CDS_RDM_LDAP_URL"]
+        ldap_url = ldap_url or current_app.config["CERN_LDAP_URL"]
```

Comment:
will this variable be picked up from the env? ( I mean if there is no "CDS_RDM" or "INVENIO" prefix)